packages/sit-onyx/src/components/OnyxDataGrid/features/index.ts

```diff
@@ -219,15 +219,41 @@ export const useDataGridFeatures = <
     header: { component: HeaderCell },
     cell: { component: (props) => String(props.modelValue) },
   };
+
+  const numberFormatter = (value: TEntry[keyof TEntry] | undefined) => {
+    if (typeof value !== "number") {
+      return;
+    }
+    const locale = injectI18n().locale;
+    const formatter = new Intl.NumberFormat(locale.value);
+
+    return formatter.format(value);
+  };
+
+  const setRenderer = (type: PropertyKey | undefined) => {
```

Comment:
```suggestion
  const setRenderer = (type?: PropertyKey) => {
```

Reply:
Why is this function needed anyway? Can't we just always define those renderers?